lms/utils/mail.py

```diff
@@ -0,0 +1,63 @@
+from functools import wraps
+
+from flask import url_for
+from flask_babel import gettext as _  # type: ignore
+from flask_mail import Message  # type: ignore
+
+from lms.lmsdb.models import User
+from lms.lmsweb import config, webapp, webmail
+from lms.models.users import generate_user_token
+
+
+def send_message(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        msg = func(*args, **kwargs)
+        if not webapp.config.get('TESTING'):
+            webmail.send(msg)
```

Comment:
We should test it works. I know it can be a bit PITA to test mails, but it has to be done ^^"

Reply:
Just ran over the instance of `webmail` with a fixture